docs/xrunjdwp.md

```diff
@@ -0,0 +1,116 @@
+﻿<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -Xrunjdwp
+
+The `-Xrunjdwp` option enables the Java&trade; Debug Wire Protocol (JDWP) agent in the VM. The JDWP agent provides a connection point for the debugger to debug a Java application in that VM.
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
+
+- You can enable the JDWP support on Checkpoint/Restore In Userspace (CRIU) restore by specifying the following command-line options pre-checkpoint:
+
+    - `-Xrunjdwp`
+    - `-agentlib:jdwp`
+    - `-agentpath:/path/to/libjdwp.so`
+
+- In OpenJ9, instead of specifying `-Xrunjdwp`, `-agentlib:jdwp`, or `-agentpath:/path/to/libjdwp.so` as command-line options pre-checkpoint, you can add these command-line options in a restore option file and specify it by using the `CRIUSupport.registerRestoreOptionsFile` API when you restore a checkpoint image.
```

Comment:
Doesnt the enablement of jdwp via `CRIUSupport.registerRestoreOptionsFile` require `-XX:+debugonrestore` @JasonFengJ9

Reply:
Yes, `jvmtiHookVMRestoreCRIUInit()` enables `jdwp` agent according to the option file settings, which is guarded by `isDebugOnRestoreEnabled(vm)` that requires `-XX:+debugonrestore`.
https://github.com/eclipse-openj9/openj9/blob/17ee5e7dd4e0b5b52321966907125d88aef13380/runtime/jvmti/jvmtiHook.c#L2050-L2051

`CRIUSupport.registerRestoreOptionsFile()` could register a restore option with `jdwp` along other command line options, but `jdwp` settings only effect with `-XX:+debugonrestore`.

Reply:
https://github.com/eclipse-openj9/openj9-docs/pull/1434#issuecomment-2438551654 - As per this comment, the following line was deleted:

> Restrictions: This option takes effect only when the -XX:+EnableCRIUSupport option and the -XX:+DebugOnRestore option are enabled.

Reply:
@tajila @JasonFengJ9 - Any changes to be made?

Reply:
In that case I think we should only mention the capabiliteis that take effect without `-XX:+debugonrestore`. So I would omit anything that talks about adding jdwp options on restore.

Reply:
@tajila So, does that mean that the title that I have used in [What's new in version 0.48.0](version0.48.md#jdwp-support-on-checkpointrestore-in-userspace-criu-restore-is-enabled)  "JDWP support on Checkpoint/Restore In Userspace (CRIU) restore is enabled" is not valid? 

Looks like I might have to delete most of the content now. Could you please let me know what content I should delete and what I should retain? Thanks!